contracts/JBController.sol

```diff
@@ -475,28 +508,31 @@ contract JBController is IJBController, JBTerminalUtility, JBOperatable, Reentra
     returns (uint256 beneficiaryTokenCount)
   {
     // Can't send to the zero address.
-    require(_reservedRate == 10000 || _beneficiary != address(0), '0x2f: ZERO_ADDRESS');
+    if (_reservedRate != MAX_TOKEN_RATE && _beneficiary == address(0)) {
+      revert BENEFICIARY_ZERO_ADDRESS();
+    }
 
     // There should be tokens to mint.
-    require(_tokenCount > 0, '0x30: NO_OP');
+    if (_tokenCount == 0) {
+      revert ZERO_TOKEN_TO_MINT();
+    }
 
     // Get a reference to the project's current funding cycle.
     JBFundingCycle memory _fundingCycle = fundingCycleStore.currentOf(_projectId);
 
     // If the message sender is not a terminal delegate, the current funding cycle must not be paused.
-    require(
-      !_fundingCycle.mintPaused() || directory.isTerminalDelegateOf(_projectId, msg.sender),
-      '0x31: PAUSED'
-    );
+    if (_fundingCycle.mintPaused() && !directory.isTerminalDelegateOf(_projectId, msg.sender)) {
+      revert MINT_PAUSED();
+    }
```

Comment:
The error message should be more descriptive. MINT_PAUSED_AND_NOT_TERMINAL_DELEGATE.